modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/html/start.jsp

```diff
@@ -120,26 +120,31 @@
 
 		<aui:input name="<%= HtmlUtil.getAUICompatibleId(ddmFormValuesInputName) %>" type="hidden" />
 
+		<aui:input name="ddmFormFieldNamespace" type="hidden" value="<%= fieldsNamespace %>" />
```

Comment:
hi @antonio-ortega , since this is only used by DL, is it possible to move this out of the DDM Taglib and keep the logic at DL?